src/styles/index.scss

```diff
@@ -1,3 +1,243 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  transition-duration: 300ms;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+}
+
+.nav__list {
+  list-style: none;
+  display: flex;
+  font-size: 12px;
+  font-weight: 500;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__link {
+  display: block;
+  line-height: 60px;
+  text-decoration: none;
+  color: #060b35;
+  text-transform: uppercase;
+  text-align: center;
+
+  &:hover {
+    color: #00acdc;
+  }
+}
+
+.nav__item:not(:first-child) {
+  margin-left: 20px;
+}
+
+.is-active {
+  position: relative;
+  color: #00acdc;
+}
+
+.is-active::after {
+  content: '';
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  border-radius: 8px;
+  height: 4px;
+  background-color: #00acdc;
+}
+
+.container {
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+  gap: 48px 46px;
+  justify-content: center;
+}
+
+@media (min-width: 488px) {
+  .container {
+    grid-template-columns: repeat(2, 200px);
+  }
+}
+
+@media (min-width: 768px) {
+  .container {
+    grid-template-columns: repeat(3, 200px);
+  }
+}
+
+@media (min-width: 1024px) {
+  .container {
+    grid-template-columns: repeat(4, 200px);
+  }
+}
+
+.card {
+  margin: auto;
+  width: 200px;
+  height: 408px;
+  border: 1px solid #f3f3f3;
+  padding-top: 32px;
+  box-sizing: border-box;
+  transition: transform 300ms;
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    text-align: left;
+    margin: auto;
+    color: rgba(6, 11, 53, 1);
+    padding: 40px 16px 0;
+
+    &-p {
+      padding: 0;
+      margin: 0;
+    }
+  }
+
+  &:hover {
+    transform: scale(1.2);
+
+    .card__title {
+      color: #34568b;
+    }
+  }
+
+  &__image {
+    background-image: url(../images/imac.jpeg);
+    background-size: 100% 100%;
+    width: 160px;
+    height: 134px;
+    margin: auto;
+    border: none;
+    padding: 0;
+  }
+
+  &__code {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    text-align: left;
+    margin: auto;
+    color: #616070;
+    padding: 4px 16px 0;
+
+    &-p {
+      padding: 0;
+      margin: 0;
+    }
+  }
+
+  &-stars {
```

Comment:
The `&-stars` selector should be nested correctly under its parent class. Ensure that the nesting is consistent with the parent class to avoid specificity issues.